recipes/kmsxx/meta.yaml

```diff
@@ -0,0 +1,74 @@
+{% set version = "0.0.0" %}
+
+package:
+  name: kmsxx-split  # not actually built, outputs define real packages
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/tomba/kmsxx.git
+  # pin to this while upstream handles https://github.com/tomba/kmsxx/issues/83
+  git_rev: 1813adae89203e15fc60344864fc2ca0d3c75e07
```

Comment:
Unfortunately we cannot accept packages into conda-forge that do not use tagged releases with semantic versioning.  Please work with the upstream repository to get tagged releases and also keep in mind that future updates also need to be from tagged releases, not git commits.

Reply:
I may be wrong, but I do not think there is any requirement for semantic versioning anywhere in conda-forge, as long as the run_exports are correctly declared?

Furthermore, if the upstream position is that they are not interested in tags, could it make sense to use calver with fixed commits? I vaguely remember some feedstock doing that, but I am currently on phone only so I can't easily search for it.

Reply:
Also please change this to use a source archive instead of `git_url`.  It should look something like:
```
url: https://github.com/tomba/kmsxx/archive/1813adae89203e15fc60344864fc2ca0d3c75e07.tar.gz
```
and a `sha256` checksum.